### PR TITLE
fix styles on hover of homepage search inputs

### PR DESCRIPTION
**Pull request by alexbowen**
no ticket

- styles fixed on focus of search input elements on homepage
- removed unused file

### CODE REVIEW

File: app/frontend/styles/components/search_panel.scss [hunk 1]
```diff
@@ -45,7 +43,6 @@
 
   .location-radius-select {
     display: inline-block;
-    float: right;
```
Comment (cpjmcquillan) [inline app/frontend/styles/components/search_panel.scss:48]: We need this to look like this 
image and not like this 
image

Comment (alexbowen) [inline app/frontend/styles/components/search_panel.scss:48]: this makes so sense alongside inline-block so removed it

Comment (cpjmcquillan) [reply app/frontend/styles/components/search_panel.scss:48]: Nevertheless, we need the top image not the bottom one. This branch regresses to the bottom image

Comment (alexbowen) [reply app/frontend/styles/components/search_panel.scss]: ah ok, i forgot this appeared, but floating is not the best way to do it and IE might have a problem with it but i havent got ie vm at the moment so will investigate when ive cleared some space on my hard drive

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**